packages/discord.js/src/structures/GuildForumChannel.js

```diff
@@ -0,0 +1,109 @@
+ * @property {boolean} moderated Whether this tag can only be added to or removed from threads
+ *  by a member with the `ManageThreads` permission
+ * @property {Snowflake} emojiId The id of a guild's custom emoji, or 0 if unset
+ * @property {string} [emojiName] The unicode character of the emoji
+ */
+
+/**
+ * @typedef {Object} DefaultReaction
+ * @property {Snowflake} emojiId The id of a guild's custom emoji, or 0 if unset
+ * @property {string} [emojiName] The unicode character of the emoji
+ */
+
+/**
+ * Represents a channel that only contains threads
+ * @extends {GuildChannel}
+ */
+class GuildForumChannel extends GuildChannel {
+  constructor(guild, data, client) {
+    super(guild, data, client, false);
+
+    /**
+     * A manager of the threads belonging to this channel
+     * @type {GuildForumThreadManager}
+     */
+    this.threads = new GuildForumThreadManager(this);
+
+    this._patch(data);
+  }
+
+  _patch(data) {
+    super._patch(data);
+    if ('available_tags' in data) {
+      /**
+       * The set of tags that can be used in this channel.
+       * @type {GuildForumTag[]}
+       */
+      this.availableTags = data.available_tags.map(tag => transformGuildForumTag(tag));
+    } else {
+      this.availableTags ??= [];
+    }
+
+    if ('default_reaction_emoji' in data) {
+      /**
+       * The emoji to show in the add reaction button on a thread in a guild forum channel
+       * @type {DefaultReaction}
+       */
+      this.defaultReactionEmoji = data.default_reaction_emoji
+        ? transformGuildDefaultReaction(data.default_reaction_emoji)
+        : null;
+    } else {
+      this.defaultReactionEmoji ??= null;
+    }
+
+    if ('default_thread_rate_limit_per_user' in data) {
+      /**
+       * The initial rate_limit_per_user to set on newly created threads in a channel.
+       * @type {number}
+       */
+      this.defaultThreadRateLimitPerUser = data.default_thread_rate_limit_per_user;
+    } else {
+      this.defaultThreadRateLimitPerUser ??= null;
+    }
+  }
+
+  /**
+   * Sets the available tags for this forum channel
+   * @param {GuildForumTag[]} availableTags The tags to set as available in this channel
+   * @param {string} [reason] Reason for changing the available tags
+   * @returns {Promise<GuildForumChannel>}
+   */
+  async setAvailableTags(availableTags, reason) {
+    await this.edit({ availableTags, reason });
+    return this;
+  }
+
+  /**
+   * Sets the default reaction emoji for this channel
+   * @param {DefaultReaction} defaultReactionEmoji The emoji to set as the default reaction emoji
+   * @param {string} [reason] Reason for changing the default reaction emoji
+   * @returns {Promise<GuildForumChannel>}
+   */
+  async setDefaultReactionEmoji(defaultReactionEmoji, reason) {
+    await this.edit({ defaultReactionEmoji, reason });
+    return this;
+  }
+
+  /**
+   * Sets the default rate_limit_per_user for new threads in this channel
```

Comment:
Where you've mentioned "rate_limit_per_user", other places in the documentation don't do this:

https://github.com/discordjs/discord.js/blob/d14d47b62fea9a57570f24195ca486886751c6fa/packages/discord.js/src/structures/interfaces/TextBasedChannel.js#L371-L379

https://github.com/discordjs/discord.js/blob/d14d47b62fea9a57570f24195ca486886751c6fa/packages/discord.js/src/managers/GuildManager.js#L95

I believe this should keep the same format.